assets/js/hooks/session.js

```diff
@@ -278,6 +281,7 @@ const Session = {
 
     this.subscriptions.forEach((subscription) => subscription.destroy());
     this.store.destroy();
+    this.history.destroy();
```

Comment:
FWIW we don't really need destroy here. In case of store we cleanup, because it uses a global id, so we don't want it to be reused in the future. The history object is scoped to this hook, so it doesn't have this problem, it's going to be garbage collected as every other part of the state :)